compiler/codegen/mli.cpp

```diff
@@ -906,13 +906,13 @@ std::string MLIContext::genServersideRPC(FnSymbol* fn) {
 std::string MLIContext::genMemCleanup(Type* t, const char* var) {
   std::string gen;
 
-  if (t == dtStringC || t == exportTypeCharPtr) {
+  if (t == dtStringC) {
```

Comment:
Do we not clean up other pointers now?

Reply:
We no longer use `c_ptr(c_char)` as a target type in interop, so that would leave `c_string` as the only pointer type with buffer-like semantics, I believe?

Reply:
Ah, okay, I'd missed that connection